sql/core/src/main/scala/org/apache/spark/sql/Dataset.scala

```diff
@@ -705,12 +701,10 @@ class Dataset[T] private[sql](
    *
    * @param right Right side of the join operation.
    * @param usingColumn Name of the column to join on. This column must exist on both sides.
-   *
-   * @note If you perform a self-join using this function without aliasing the input
-   * [[DataFrame]]s, you will NOT be able to reference any columns after the join, since
-   * there is no way to disambiguate which side of the join you would like to reference.
-   *
-   * @group untypedrel
+    * @note If you perform a self-join using this function without aliasing the input
+   *        [[DataFrame]]s, you will NOT be able to reference any columns after the join, since
+   *        there is no way to disambiguate which side of the join you would like to reference.
+    * @group untypedrel
```

Comment:
Oh, no. I mean some changes here such as

```diff
-   *
-   * @group action
+    * @group action
```

seems not related with what the PR tries to proposes.